posts/inside-rust/2022-04-13-imposter-syndrome.md

```diff
@@ -0,0 +1,108 @@
+prolific contributors lifting mountains by themselves, it's everyone working
+together that brought us to where we are today. We all make mistakes. The
+project has layer[^4] after layer[^5] of safeguards to make sure we have a
+chance to catch and fix them before they affect our users. These incidents are
+unavoidable, expected, and honestly fine! This is the most fundamental
+philosophy of both the Rust language and the Rust project: we don't think it's
+sufficient to build robust systems by only including people who don't make
+mistakes; we think it's better to provide tooling and process to catch and
+prevent mistakes. It isn't an accident that our motto is "A language empowering
+everyone to build reliable and efficient software." We want people to feel
+empowered to make changes they're not 100% confident in, to make mistakes, to
+learn, and to grow within the Rust project. This is how all of us got to where
+we are today!
+
+So, if you look up to people within the Rust project, if the work we do here
+interests you, if you have always wanted to contribute, and _especially_ if you
+already have contributed, I want you to know that you're one of the people we
+want to apply for [Rust Foundation grants and
+fellowships](https://foundation.rust-lang.org/grants/). You're one of the
+people we want to eventually see join teams. You're not an imposter, and I want
+you to know that I really look forward to seeing you around the project.
+
+---
+
+*To help reinforce and normalize this, I've gathered a list of times when
+current or past project members have struggled with imposter syndrome, have
+made mistakes, have had to ask "basic" questions, and similar experiences that
+will hopefully help set more reasonable expectations for new and old
+contributors across the project.*
+
+* [Jane Lusby](https://github.com/yaahc/): "I frequently struggle with imposter
+  syndrome and feeling like I don't get as much done as my peers. When I do all
+  of my work based off of notifications I completely lose track of what I've
+  done and end up starving the tasks I wanted to work on. I'm learning to set
+  reasonable expectations for myself, getting better at managing distractions,
+  and being intentional about when I respond to github/zulip notifications
+  which helps me with keeping track of what I've done and making steady
+  progress on my priorities."
+* [Josh Triplett](https://github.com/joshtriplett/): "I didn't fully understand
+  `Pin` until I read fasterthanlime's ["Pin and
+  suffering"](https://fasterthanli.me/articles/pin-and-suffering) blog post and
+  I gave a talk in 2016 where my [most important
+  point](https://www.youtube.com/watch?v=U8Gl3RTXf88#t=24m40s) was that people
+  erroneously believe that you have to be an expert to write an RFC or change
+  Rust, but that I wasn't, and you don't need to be one either."
+* [Ralf Jung](https://github.com/ralfjung): "I am still surprised anyone is
+  taking Miri and Stacked Borrows seriously."
+
```

Comment:
Here's a section from me that I hope fits alright :)

```suggestion
* [Forest Anderson](https://github.com/angelonfira/): "As someone who just
  learned last week what `dyn` does, it still amazes me that I have something to
  give as a team lead. I was immersed in Rust communities by writing weekly
  blogs about Veloren (I took this on because I didn't know enough to contribute
  code), which lead to helping with the Rust Gamedev newsletter, which led me to
  helping to run the Cross Team Collaboration Fun Times meetup!"

```